tracer/src/Datadog.Tracer.Native/tracer_tokens.cpp

```diff
@@ -152,13 +153,33 @@ HRESULT TracerTokens::EnsureBaseCalltargetTokens()
     {
         return hr;
     }
+
+    
+
     // *** Ensure Datadog.Trace.ClrProfiler.CallTarget.CallTargetBubbleUpException type ref, might not be available if tracer version is < 2.22
-    if (bubbleUpExceptionTypeRef == mdTypeRefNil)
+    if (trace::profiler->IsCallTargetBubbleUpExceptionTypeAvailable() && bubbleUpExceptionTypeRef == mdTypeRefNil)
     {
         const ModuleMetadata* module_metadata = GetMetadata();
-        module_metadata->metadata_emit->DefineTypeRefByName(profilerAssemblyRef,
+        const auto defined_call_target_bubble_up_exception_type = module_metadata->metadata_emit->DefineTypeRefByName(profilerAssemblyRef,
```

Comment:
nit: personally I don't think `defined_call_target_bubble_up_exception_type` is the best name for an HRESULT value, without the context is hard to tell it's an HRESULT.

Reply:
changed  in [0721e36](https://github.com/DataDog/dd-trace-dotnet/pull/5034/commits/0721e36e224c9d302ddc56c42a4b61bf41f954c5)